docs/general/attributes.md

```diff
@@ -410,34 +414,7 @@ system. It is expected this information would be propagated unchanged from node-
 using the Baggage mechanism. These attributes should not be used to record system-to-system
 authentication attributes.
 
-Examples of where the `enduser.id` value is extracted from:
-
-| Authentication protocol | Field or description            |
```

Comment:
I would prefer to keep this table - it actually explains what the `enduser.id` is - we don't provide an explanation like this anymore. Is this information still accurate and can it be used to capture `enduser.id` ?

Reply:
`enduser.id` has been replaced with 'enduser.pseudo.id`. i don't think it makes sense to keep it.